### PR TITLE
LPS-96616

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java [hunk 1]
```diff
@@ -1759,26 +1771,29 @@ public void testBuildTemplatePortlet70() throws Exception {
...
 			gradleProjectDir, "build.gradle",
 			"apply plugin: \"com.liferay.plugin\"",
 			_DEPENDENCY_PORTAL_KERNEL + ", version: \"2.0.0");
 		_testContains(
 			gradleProjectDir,
 			"src/main/java/foo/test/constants/FooPortletKeys.java",
-			"public class FooPortletKeys");
+			"public class FooPortletKeys",
+			"public static final String FOO",
+			"\"foo.test_FooPortlet\";");
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java]: hey Lawrence, this should be `foo_test_FooPortlet` to match the portal parameters.  So it seems the velocity is not working as expected here: `${package.replaceAll("\\.", "_")}_${className}Portlet`

Comment (gamerson) [reply modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java]: I just debuged it and it should be `"${package.replaceAll('\.', '_')}_${className}Portlet";`

File: modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/src/main/java/constants/__className__PortletKeys.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
  */
 public class ${className}PortletKeys {
 
-	public static final String ${className} = "${className.toLowerCase()}";
+	public static final String ${className.toUpperCase()} =
+		"${package.replaceAll("\\.", "_")}_${className}Portlet";
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/src/main/java/constants/__className__PortletKeys.java]:
```suggestion
		"${package.replaceAll('\.', '_')}_${className}Portlet";
```

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/petershin/liferay-portal/pull/541. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**